include/bouncer.h

```diff
@@ -305,6 +306,11 @@ struct PgPool {
 	 * Clients that sent cancel request, to cancel another client its query.
 	 * These requests are waiting for a new server connection to be opened,
 	 * before the request can be forwarded.
+	 *
+	 * This is a separate list from waiting_client_list, because we want to
```

Comment:
Is this related to this patch?

Reply:
I extracted this small change and moved it to: https://github.com/pgbouncer/pgbouncer/pull/1016